src/phpDocumentor/ConfigurationFactory/PhpDocumentor3.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\ConfigurationFactory;
+
+/**
+ * phpDocumentor3 strategy for converting the configuration xml to an array.
+ */
+final class PhpDocumentor3 implements Strategy
+{
+    /**
+     * The path to the xsd that is used for validation of the configuration file.
+     *
+     * @var string
+     */
+    private $schemaPath;
+
+    /**
+     * Initializes the PhpDocumentor3 strategy.
+     *
+     * @param string $schemaPath
+     */
+    public function __construct($schemaPath)
+    {
+        if ($schemaPath === '') {
+            $schemaPath = __DIR__ . '../../../../data/xsd/phpdoc.xsd';
```

Comment:
there should be a '/' at the start of the relative path string; without it it will concatenate the directory name with .. and there will be no slash in between